python/modules/IcePy/Types.h

```diff
@@ -563,8 +563,6 @@ namespace IcePy
     {
     public:
         ValueWriter(PyObject*, ObjectMap*, const ValueInfoPtr&);
-        ~ValueWriter();
```

Comment:
Can ValueWriter be released from the C++ thread?

Reply:
I don't think it can.

My understanding is ValueWriter is a transient object used when marshaling a payload. This all occurs in the Python thread, with the GIL locked.